.eslintrc.json

```diff
@@ -28,7 +28,11 @@
     "react/jsx-props-no-spreading": "off",
     "jsx-a11y/click-events-have-key-events": "off",
     "jsx-a11y/no-noninteractive-element-interactions": "off",
-    "jsx-a11y/no-static-element-interactions": "off"
+    "jsx-a11y/no-static-element-interactions": "off",
+    "import/no-unresolved": "off",
```

Comment:
nope

Reply:
Resolve the "import/no-unresolved": "off"